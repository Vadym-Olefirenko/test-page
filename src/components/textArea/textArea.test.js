import React from 'react';
import TextArea from '../textArea/textarea';
import {create, act} from 'react-test-renderer';
import { configure, shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('Testing textarea value', () => {
    test('Value', () => {
        const textFieald = shallow(<TextArea />);
        
        expect(textFieald.debug()).toMatchSnapshot();
      
    });

    test('Value', () => {
        const textFieald = shallow(<TextArea value="aaaaa"/>);
        
        expect(textFieald.debug()).toMatchSnapshot();
       
    });

});

