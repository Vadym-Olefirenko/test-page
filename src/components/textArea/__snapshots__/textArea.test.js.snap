// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing textarea value Value 1`] = `"<textarea value={[undefined]} onChange={[undefined]} name=\\"text\\" placeholder=\\"Введіть текст або  \\" />"`;

exports[`Testing textarea value Value 2`] = `"<textarea value=\\"aaaaa\\" onChange={[undefined]} name=\\"text\\" placeholder=\\"Введіть текст або  \\" />"`;
